extensions/localserver-cli-command/src/client-scripts/LoadTemplateSafe.js

```diff
@@ -28,7 +28,7 @@
 	});
 
 	function copyProperties(source, dest) {
-		for (var property in source) {
+		for (const property in source) {
```

Comment:
Is it going to be alright to change from var to const in this client-script? Is my only concern in the whole pull request. Just because maybe this code is run in browser.

Reply:
Good catch, I'll revert it!